packages/block-library/src/editor.scss

```diff
@@ -53,7 +54,7 @@
 
 // Provide every block with a default base margin. This margin provides a consistent spacing
 // between blocks in the editor.
-.editor-styles-wrapper [data-block] {
+.editor-styles-wrapper .block-editor-block-list__block.has-spacing > .block-editor-block-list__block-edit > [data-block] {
```

Comment:
Is this necessary? I suspect this will cause a lot of editor styles to break because it adds needless extra specificity to editor styles.